test/src/test/java/hudson/util/ArgumentListBuilder2Test.java

```diff
@@ -92,12 +92,10 @@ public void ensureArgumentsArePassedViaCmdExeUnmodified() throws Exception {
     }
 
     public String echoArgs(String... arguments) throws Exception {
-        String testHarnessJar = Class.forName("hudson.util.EchoCommand")
+        String testHarnessJar = new File(Class.forName("hudson.util.EchoCommand")
```

Comment:
Could also use `Paths.get(…)`, or `Which.jarFile`, but this seems to work well enough.